jerry-core/parser/js/byte-code.h

```diff
@@ -665,6 +671,9 @@ typedef enum
   CBC_CODE_FLAGS_ARROW_FUNCTION = (1u << 7), /**< this function is an arrow function */
   CBC_CODE_FLAGS_STATIC_FUNCTION = (1u << 8), /**< this function is a static snapshot function */
   CBC_CODE_FLAGS_DEBUGGER_IGNORE = (1u << 9), /**< this function should be ignored by debugger */
+#ifndef CONFIG_DISABLE_ES2015_CLASS
+  CBC_CODE_FLAGS_CLASS_CONSTRUCTOR = (1u << 10), /**< this function is a class constructor */
```

Comment:
I would just use CBC_CODE_FLAGS_CONSTRUCTOR since we might use it later for other purposes.